spec/schemas/_common.yaml

```diff
@@ -1814,39 +1814,39 @@ components:
           description: The total amount of time for all shard scroll operations.
           $ref: '#/components/schemas/Duration'
         scroll_time_in_millis:
-          description: The total amount of time for all shard scroll operations, in milliseconds.
+          description: The total amount of time for all shard scroll operations, in milliseconds. 
```

Comment:
"The total amount of time taken to complete all shard scroll operations, in milliseconds"?